src/main/java/jenkins/plugins/clangscanbuild/publisher/ClangScanBuildPublisherDescriptor.java

```diff
@@ -41,10 +40,7 @@ public String getDisplayName() {
 
 	@Override
 	public boolean isApplicable( @SuppressWarnings("rawtypes") Class<? extends AbstractProject> jobType ){
-		if( !FreeStyleProject.class.isAssignableFrom( jobType ) ){
-			System.err.println( "Clang scan-build ERROR: Expected FreeStyleProject but was: " + jobType + " at Publisher Descriptor" );
-		}
-		return AbstractProject.class.isAssignableFrom( jobType );
+		return true;
```

Comment:
All the jobs that extends AbstractProject are allowed so we just need to return true.